### ISSUE TITLE
Files step3-5

### ISSUE THREAD

**Issue by MarcoSal** [yumorishita/LiCSBAS#272]
An issue related to step3-5 is not clear to me.
Once step 3 has been done, the generated files, where are they used? I can't figure it out from the guide.

**Comment by yumorishita** [comment]
I recommend using batch_LiCSBAS.sh. You don't need to consider the input/output file locations.

**Comment by MarcoSal** [comment]
I need to carry out the processing step by step, change the variables and check their impact on the final result...

**Comment by MarcoSal** [comment]
> An issue related to step3-5 is not clear to me. Once step 3 has been done, the generated files, where are they used? I can't figure it out from the guide.

any help?

**Comment by yumorishita** [comment]
The usage for each script may be helpful:
https://github.com/yumorishita/LiCSBAS/blob/master/bin/LiCSBAS03op_GACOS.py
https://github.com/yumorishita/LiCSBAS/blob/master/bin/LiCSBAS04op_mask_unw.py
https://github.com/yumorishita/LiCSBAS/blob/master/bin/LiCSBAS05op_clip_unw.py

**Comment by yumorishita** [comment]
Yes, but the order of step03-05 can be changed (e.g., output of step05 can be input of step03).
You can change the order [here](https://github.com/yumorishita/LiCSBAS/blob/master/batch_LiCSBAS.sh#L32).

**Comment by MarcoSal** [comment]
excellent.
Thank you.
close...
